futaba/sql/models/settings.py

```diff
@@ -156,8 +202,18 @@ def __init__(self, sql, meta):
                 name="special_role_jail_not_member_check",
             ),
         )
+        self.tb_tracking_blacklists = Table(
+            "tracking_blacklists",
+            meta,
+            Column(
+                "guild_id", BigInteger, ForeignKey("guilds.guild_id"), primary_key=True
+            ),
+            Column("type", Enum(TrackingBlacklistType), primary_key=True),
```

Comment:
Is there a difference between `primary_key`ing all columns vs just adding a `UniqueConstraint`? imo the latter looks better, but I don't think it matters.

Reply:
afaik the only difference is that using multiple primary_key's results in a ddl for `ADD CONSTRAINT ... PRIMARY KEY ...;` whereas a unique constraint generates `ADD CONSTRAINT ... UNIQUE CONSTRAINT ...;` in postgres there's no difference as PRIMARY KEY is shorthand for UNIQUE NOT NULL.